src/jit/importer.cpp

```diff
@@ -1074,14 +1074,16 @@ GenTreePtr Compiler::impAssignStruct(GenTreePtr     dest,
 
 /*****************************************************************************/
 
-GenTreePtr Compiler::impAssignStructPtr(GenTreePtr      dest,
+GenTreePtr Compiler::impAssignStructPtr(GenTreePtr      destAddr,
                                         GenTreePtr      src,
                                         CORINFO_CLASS_HANDLE    structHnd,
                                         unsigned        curLevel,
                                         GenTreePtr    * pAfterStmt,  /* = NULL */
                                         BasicBlock    * block        /* = NULL */
                                        ) 
 {
+    GenTreePtr dest = nullptr;
```

Comment:
Actually, that is an intended part of the refactoring. With 1st class structs, we will be creating an actual assignment, so what we really want is a `dest` not a `destAddr`.